src/kinvey.d.ts

```diff
@@ -544,6 +544,7 @@ interface RequestOptions {
   properties?: Properties;
   timeout?: number;
   useDeltaFetch?: boolean;
+  version?: String;
```

Comment:
This change should've been in my previous typescript PR #235 . I did not realize we had two copies of the ts definitions in the same file.